wait-for-it.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
```

Comment:
This script actually doesn't work for mac so would be cool to fix it or replace it by something else:
`timeout: command not found`

Reply:
Would it be ok to put [this](https://github.com/vishnubob/wait-for-it/issues/15) workaround in `readme` if using macOS? Or should I search for an alternative?

Reply:
I guess that's fine